RoboticonColony/Assets/Code/Roboticon.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// A Roboticon. Holds all information about the roboticon and facilitates upgrading said roboticon. 
+/// </summary>
+public class Roboticon
+{
+    public Tile CurrentTile { get; private set; }
+
+    public AbstractPlayer Owner { get; private set; }
```

Comment:
I'm not sure that we really need a Roboticon to have a owner, since it's never used in the class. A Roboticon has to have a current tile, which has an owner, to even exist, so we can probably go off the owner in CurrentTile.